index.Rmd

```diff
@@ -356,7 +357,7 @@ All pixel values of the original masked raster are then changed to a value of 1,
 masknew3=masknew/masknew
 ``` 
 
-A dataframe is then compiled which includes the name of the County, the total number of pixels within that county, the total number of mixed/deciduous forest pixels within the county, and the percent of pixels which are mixed/deciduous.
+A dataframe is then compiled which includes the name of the County, the total number of pixels within that county, the total number of mixed/deciduous forest pixels within the county, and the percent of pixels which are mixed/deciduous. 
```

Comment:
I loved how you split up the code with short explanations in this section. It made it super easy to follow!

Reply:
Thanks!